### PR TITLE
adds Charles to leadership page

### PR DESCRIPTION
**Pull request by SBBCornell**
I have taken a first stab at a brief bio. Please feel free to edit, or suggest edits. Here is a screenshot:
Screen Shot 2022-06-27 at 12 00 26 PM

### CODE REVIEW

**Review by jimentwood — APPROVED** [review]
looks good

**Comment by SBBCornell** [comment]
Here is the leadership page with all changes:
Screen Shot 2022-06-27 at 1 54 24 PM

**Comment by SBBCornell** [comment]
screenshot showing all changes to leadership page. Brian C. is removed (discussed between Brian and Charles), degree mentions are standardized, some other minor edits. 
Screen Shot 2022-06-27 at 2 07 01 PM

File: about/people/leadership_team.md [hunk 1]
```diff
@@ -22,40 +16,44 @@ people:
...
+      Ramin is a computer science professor at Cornell Tech and president and founder of the Computer Vision Foundation. His research focuses on computer vision and its applications, especially in medical imaging. As arXiv faculty director, Ramin guides arXiv’s strategic vision and technological modernization with input from the global research and scholarly communications communities. He holds a Ph.D. in Computer Science and Mathematics from Stanford University.
+  charles:
+    name: Charles Frankston
+    position: Technical Director
+    thumbnail: images/charles.jpg
+    bio: |
+      Charles is focused on helping ready arXiv for it's next 30 years of stable support of open science and will shepherd arXiv's move to the cloud. He brings his wealth of expertise in a wide range of platforms and languages, systems architecture, and deep commitment to data privacy. Charles holds a B.S. in Computer Science and Engineering from Massachusetts Institute of Technology.
```
Comment (cbf66) [inline about/people/leadership_team.md:42]: from Massachusetts Institute of Technology. -> from *the* Massachusetts Institute of Technology.

**Review by cbf66 — CHANGES_REQUESTED** [review]
Either "from MIT" or from "the Massachusetts Institute of Technology". 

And I suppose the latter. There may be someone on the planet who doesn't know what MIT is (or thinks it's the Turkish Intelligence Organization)

**Pull request merged**